sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -1454,6 +1454,7 @@ public virtual Response DeleteModel(string modelId, CancellationToken cancellati
         /// <param name="query">The query string, in SQL-like syntax.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The pageable list <see cref="AsyncPageable{T}"/> of query results.</returns>
+        /// <typeparam name="T">The type to deserialize the component to.</typeparam>
```

Comment:
"... to deserialize the result to." since it won't be a component, and it may not be a digital twin each time either